runelite-client/src/main/java/net/runelite/client/plugins/motherlode/MotherlodePlugin.java

```diff
@@ -137,17 +139,39 @@ MotherlodeConfig getConfig(ConfigManager configManager)
 		return Arrays.asList(overlay, rocksOverlay, motherlodeSackOverlay, motherlodeGemOverlay);
 	}
 
+	private void refreshSackValues()
+	{
+		curSackSize = client.getVar(Varbits.SACK_NUMBER);
+		boolean sackUpgraded = client.getVar(Varbits.SACK_UPGRADED) == 1;
+		maxSackSize = sackUpgraded ? SACK_LARGE_SIZE : SACK_SIZE;
+	}
+
 	@Override
 	protected void startUp()
 	{
 		inMlm = checkInMlm();
+
+		if (inMlm)
+		{
+			refreshSackValues();
+		}
 	}
 
 	@Override
 	protected void shutDown() throws Exception
 	{
 		veins.clear();
 		rocks.clear();
+
+		// Reset session stats
+		session.reset();
```

Comment:
It may be best to either reset this on startup or make it non final and make a new session on startup.

Reply:
Updated so it does it on startup rather than shutdown